README.md

```diff
@@ -1,145 +1,42 @@
 # Amundsen
 
-Amundsen is a data discovery web portal. It leverages a separate [search service](https://github.com/lyft/amundsensearchlibrary) for allowing users to search for data resources, and a separate [metadata service](https://github.com/lyft/amundsenmetadatalibrary) for viewing and editing metadata for a given resource. It is a Flask application with a React frontend.
+Amundsen is a data discovery portal to make data more discoverable. The project named after Norwegian explorer [Roald Amundsen](https://en.wikipedia.org/wiki/Roald_Amundsen) collects and displays
+metadata (owner, stats, which Airflow DAG generates, source file, etc.) about various data resources such as Hive/Redshift tables, 
```

Comment:
nit: I'd also rather explicitly state something like "Amundsen supports Hive and Redshift tables." so that no one assumes we have wider support than we currently do.